docs/learn/fundamentals/transactions/unique-source-for-operation.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "When Operation Source Accounts differ from that of the Parent Transaction"
+sidebar_label: Unique Source Accounts for Operations
+description: "Learn"
+sidebar_position: 40
+---
+
+# When Operation Source Accounts differ from that of the Parent Transaction
+
+Let's review the unique case where an operation may have a source account different from that of the parent transaction envelope.
+
+There are 2 main situations where this may happen:
+
+1. Operations that affect more than one account
+2. Operations with a high threshold weight
+
+## Operations that Affect multiple Accounts
+
+If a transaction has operations with multiple source accounts, it will require the source account signature for each operation.
+
+One case where this happens is with specialized transactions such as [Sponsored Reserves](../../encyclopedia/transactions-specialized/sponsored-reserves) where a sponsoring account agrees to pay the base reserves for a sponsored account.
+
+In this example, there are 2 sponsoring accounts, `S1` and `S2`, that sponsor trustlines for Account `A`.
+
+```ts
+//
+// 2. Both S1 and S2 sponsor trustlines for Account A for different assets.
+//
+let aAccount = await server.loadAccount(A.publicKey());
+let tx = new TransactionBuilder(aAccount, { fee: BASE_FEE })
+  .addOperation(
+    Operation.beginSponsoringFutureReserves({
+      source: S1.publicKey(),
+      sponsoredId: A.publicKey(),
+    }),
+  )
+  .addOperation(
+    Operation.changeTrust({
+      asset: usdcAsset,
+      limit: "5000",
+    }),
+  )
+  .addOperation(Operation.endSponsoringFutureReserves({}))
+
```

Comment:
i'd probably get rid of this blank line here

Reply:
oh, i see now. we're separating the two sponsorship sandwiches with the line.

i could go either way